doc/source/gallery/geopandas_rasterio_sample.ipynb

```diff
@@ -0,0 +1,321 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "# Using GeoPandas with Rasterio to sample point data\n",
+    "\n",
+    "This example shows how to use GeoPandas with Rasterio.  [Rasterio](https://rasterio.readthedocs.io/en/latest/index.html) is a package for reading and writing raster data.\n",
+    "\n",
+    "In this example a set of vector points is used to sample raster data at those points.\n",
+    "\n",
+    "The raster data used is Copernicus Sentinel data 2018 for Sentinel data.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import geopandas\n",
+    "import rasterio\n",
+    "import matplotlib.pyplot as plt\n",
+    "from shapely.geometry import Point"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Create example vector data\n",
+    "=============================\n",
+    "\n",
+    "Generate a geodataframe from a set of points\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create sampling points\n",
+    "points = [Point(625466, 5621289), Point(626082, 5621627), Point(627116, 5621680), Point(625095, 5622358)]\n",
+    "gdf = geopandas.GeoDataFrame([1, 2, 3, 4], geometry=points)"
```

Comment:
```suggestion
    "gdf = geopandas.GeoDataFrame([1, 2, 3, 4], geometry=points, crs=32630)"
```
If possible, try to avoid the creation of gdf without a CRS.